### DISCUSSION TITLE
is quik still alive ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [octoshrimpy/quik#145] [General]
Is this project still alive ?

**Comment by Zumux** [comment]
I'd also like to know this.

**Comment by octoshrimpy** [comment]
Hi! Quik is still alive. I'm in the process of moving across the states for a new job (and towing my family along!). My PC is still in a box that only got delivered to the new place earlier today(!), so there will be updates coming down the pipe soon.

**Comment by Zumux** [comment]
Glad to hear!
